app/imports/components/ActionItems/ActionItemsBoard/ActionItemsBoard.jsx

```diff
@@ -193,14 +195,18 @@ class ActionItems extends React.Component {
                     open={openSnackbar}
                     message="New action item has been added!"
                     autoHideDuration={4000}
-                    onRequestClose={() => closeSnackBar(sprintId, onData, handlers, wrappedData)}
+                    onRequestClose={() => closeSnackBar(
+                        sprintId, onData, handlers, hideButton, wrappedData,
+                    )}
                 />
 
                 <Snackbar
                     open={openToggleSnackbar}
                     message="Changes saved!"
                     autoHideDuration={4000}
-                    onRequestClose={() => closeSnackBar(sprintId, onData, handlers, wrappedData)}
+                    onRequestClose={() => closeSnackBar(
```

Comment:
Also you shouldn't take `closeSnackBar` from props, add this method to component.